package.json

```diff
@@ -92,4 +92,4 @@
     "node": ">=4.0.0",
     "npm": ">=3.0.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
@hedss what's the story on fixing this? I thought it had already been done ages back